### PR TITLE
fix(flags): better redistribute experience

### PR DESCRIPTION
**Pull request by dmarticus**
## Problem

Per @raquelmsmith's feedback [here](https://github.com/PostHog/posthog/pull/25580#discussion_r1800345027), I redid the `redistribute` experience for the flags UI to now use a button with a tooltip instead of the word "redistribute" on a button. I also changed the percentage rollout input boxes to have the `%` suffix; it looks better IMO.

Worked with Cory to use a scale balance icon for this thing, since I figured that was a good visual indicator for rebalancing variant distributions.

## Changes

Demo: https://www.loom.com/share/9ccb4e1e8303434cbdc1daca86a11855

### CODE REVIEW

File: frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelExperimentFeatureFlag.tsx [hunk 1]
```diff
@@ -99,8 +100,11 @@ export const SidePanelExperimentFeatureFlag = (): JSX.Element => {
                             title: (
                                 <div className="flex items-center justify-between space-x-2">
                                     <span>Rollout Percentage</span>
-                                    <LemonButton type="secondary" size="xsmall" onClick={distributeVariantsEqually}>
-                                        Redistribute
+                                    <LemonButton
+                                        onClick={distributeVariantsEqually}
+                                        tooltip="Redistribute variant rollout percentages equally"
+                                    >
+                                        <IconBalance />
```
Comment (dmarticus) [inline frontend/src/layout/navigation-3000/sidepanel/panels/SidePanelExperimentFeatureFlag.tsx:107]: looks like this
image

File: frontend/src/scenes/feature-flags/FeatureFlag.tsx [hunk 2]
```diff
@@ -952,7 +952,12 @@ function FeatureFlagRollout({ readOnly }: { readOnly?: boolean }): JSX.Element {
                             </div>
                             <div className="col-span-4 flex items-center gap-1">
                                 <span>Rollout</span>
-                                <LemonButton onClick={distributeVariantsEqually}>(Redistribute)</LemonButton>
+                                <LemonButton
+                                    onClick={distributeVariantsEqually}
+                                    tooltip="Redistribute variant rollout percentages equally"
+                                >
+                                    <IconBalance />
+                                </LemonButton>
```
Comment (dmarticus) [inline frontend/src/scenes/feature-flags/FeatureFlag.tsx:960]: looks like this
image

Comment (raquelmsmith) [reply frontend/src/scenes/feature-flags/FeatureFlag.tsx:960]: If I'm gonna nitpick I'd still bump it over to line up with the right side of the column, but if not then I think this is fine 👍

**Review by raquelmsmith — APPROVED** [review]

File: frontend/src/scenes/feature-flags/FeatureFlag.tsx (hunk 2 above)
Comment (dmarticus) [reply frontend/src/scenes/feature-flags/FeatureFlag.tsx]: nitpick addressed!

**Pull request merged**